Lib/idlelib/pyparse.py

```diff
@@ -224,15 +196,13 @@ def set_lo(self, lo):
         if lo > 0:
             self.code = self.code[lo:]
 
-    # Build a translation table to map uninteresting chars to 'x', open
-    # brackets to '(', close brackets to ')' while preserving quotes,
-    # backslashes, newlines and hashes. This is to be passed to
-    # str.translate() in _study1().
-    _tran = {}
-    _tran.update((ord(c), ord('(')) for c in "({[")
-    _tran.update((ord(c), ord(')')) for c in ")}]")
-    _tran.update((ord(c), ord(c)) for c in "\"'\\\n#")
-    _tran = StringTranslatePseudoMapping(_tran, default_value=ord('x'))
+    # Map1 maps uninteresting chars to 'x', open brackets to '(',
+    # close brackets to ')', and preserves quotes,  backslashes,
+    # newlines and hashes.  Used for str.translate() in _study1().
+    map1 = ParseMap()
```

Comment:
What is the reason of renaming `_tran` to `map1`?

Reply:
It reads better to me.  The object is a map for _study1.  'tran' is a prefix for numerous words, often verbs, not a word in itself.

Reply:
This is a matter of style, but to me, `_tran` looks more semantic. It refers to the purpose of it (a translation table). `map1` refers only to its type. This is like naming the integer variable "int1" instead of "cnt" or "col".